docs/references/system_requirements/global.md

```diff
@@ -3,40 +3,34 @@ sidebar_label: "Global System Requirements"
 title: ""
 ---
 
-## Kubernetes Cluster Requirements
+## System Requirements
 
-For the Epinio server, and related deployments we recommend to consider the following resources:
+Please consider the following system requirements to be minimal, additional resources may be required for additional workload (apps).
 
-- x86_64, ARM32, or ARM64 architecture
-- Supported operating systems
-  - Linux
-    - [K3s](../../howtos/install_epinio_on_k3s.md)
-    - [K3d](../../howtos/install_epinio_on_k3d.md)
-    - [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)
-    - [RKE2](../../howtos/install_epinio_on_rke.md)
-  - Windows
-    - [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)
-  - MacOS
-    - [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)
-- Kubernetes versions 1.20 .. 1.23
-- 2-4 VCPUs
-- 8GB RAM (system memory + 4GB)
-- 10GB Disk space (system disk + 5GB)
+| Component | Description |
+| --- | --- |
+| OS / Kubernetes *) | Linux: [RKE2](../../howtos/install_epinio_on_rke.md), [K3s](../../howtos/install_epinio_on_k3s.md),   [K3d](../../howtos/install_epinio_on_k3d.md), [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)<br/>Windows, MacOS: [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md) |
```

Comment:
This is probably nicer:

| Component | Description |
| --- | --- |
| OS / Kubernetes <sup>1</sup> | Linux



```suggestion
| OS / Kubernetes <sup>1</sup> | Linux: [RKE2](../../howtos/install_epinio_on_rke.md), [K3s](../../howtos/install_epinio_on_k3s.md),   [K3d](../../howtos/install_epinio_on_k3d.md), [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md)<br/>Windows, MacOS: [Rancher Desktop](../../howtos/install_epinio_on_rancher_desktop.md) |
```